api/src/main/resources/messages_es.properties

```diff
@@ -0,0 +1,11 @@
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+# the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#
+# Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+# graphic logo is a trademark of OpenMRS Inc.
+#
+
+queue.entry.duplicate.patient=Patient already in the queue
```

Comment:
Either provide reasonable default translations in the appropriate language or remove the non-english translation files from the module.  They can always be translated downstream via Initializer.